src/loader.ts

```diff
@@ -58,7 +58,7 @@ function processResource(
   // deterministic.
   context.cacheable(true);
 
-  const options: LoaderOptions = getOptions(context) || {};
+  const options: LoaderOptions = Object.assign({}, getOptions(context));
```

Comment:
Can you swap this with an object spread?

```ts
const options: LoaderOptions = { ...getOptions(context) };
```

Reply:
Scratch that.

Reply:
🙏